build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
 //	implementation 'org.apache.camel.springboot:camel-amqp-starter:3.3.0'
 	implementation 'org.springframework:spring-jms:5.2.6.RELEASE'
 	implementation 'org.apache.qpid:qpid-jms-client:0.51.0'
+	implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.10'
```

Comment:
should we use guava or commons-lang3 but not both?

Reply:
removed